charts/gitlab-ci-pipelines-exporter/values.yaml

```diff
@@ -2,6 +2,8 @@
 # replicas -- amount of desired pod(s) replica(s)
 replicas: 1
 
+namespace: default
```

Comment:
I believe it would be interesting to have a template to default to `.Release.Namespace` instead of default.

Reply:
Yeah, agree